components/Proposals/ProposalPreview.tsx

```diff
@@ -0,0 +1,54 @@
+import { useRouter } from 'next/navigation';
+import Markdown from 'react-markdown';
+import { Card, Group, Stack, Text, UnstyledButton } from '@mantine/core';
+import { useExplorerConfiguration } from '@/hooks/useExplorerConfiguration';
+import { StateBadge } from './StateBadge';
+import { Proposal } from '@/lib/types';
+import ExternalLink from '../ExternalLink';
+import { shortKey } from '../../lib/utils';
+
+export function ProposalPreview({ proposal }: { proposal: Proposal }) {
+  const router = useRouter();
+  const { generateExplorerLink } = useExplorerConfiguration();
+
+  return (
+    <UnstyledButton onClick={() => router.push(`/proposal?id=${proposal.account.number}`)}>
```

Comment:
I'm not a fan of making the whole proposal element clickable, especially since the proposal's content already contains several links and it's a bit confusing what clicking the proposal effectively does

Reply:
I'm not a fan of crowding the ui with buttons, don't know how we should merge our different design views, maybe on @R-K-H 🙃

Reply:
Hmmmmm.... I'll think on it.